README.md

```diff
@@ -44,4 +45,4 @@ As we've been working, we've come up with a bunch of questions that we wanted to
 
 ### Questions?  Feedback?
 
-Let us know in the [forums](https://discuss.opendistrocommunity.dev/).
+Let us know in the [issue](https://github.com/opensearch-project/OpenSearch/issues/new/choose).
```

Comment:
'Let us know in the issue' doesn't make sense.


Perhaps something like 'Found a bug or need to leave feedback? Make sure and file an [issue]. Questions and discussions on this project can be found on the [project forums]'

Reply:
Sure. Updating it!

Reply:
For forums, we already have a section: https://github.com/opensearch-project/OpenSearch#watch-the-forum-because-there-are-a-bunch-of-things-we-want-to-talk-about

Wouldn't it be redundant? What do you think?

Reply:
It's no different than status quo - we have dual links to the forums now. TBH, I think we should drop the "watch the forums..." section as beta moves us into a more task-oriented phase of the project. That's a bigger change that at hand.

The point in having the forums down here is that we really don't want people asking questions in the issues.

Reply:
Sounds good. I'll leave it for now and then we can remove it once we are moving on to the next phase.